### PR TITLE
Additions KubeCon EU 2018 Blog post/ OpenFaaS NATS Operator

### PR DESCRIPTION
**Pull request by gcolliso**
Added KubeCon EU 2018 recap blog post (blog)
Added OpenFaaS NATS connector (download)

### CODE REVIEW

File: content/blog/kubecon_eu_2018.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+We are thankful to the Cloud Native Computing Foundation and all those who participated in KubeCon. We sincerely appreciate the community's interest in NATS and know our time spent in Copenhagen was successful to further education about NATS. We look forward to seeing you in December at [KubeCon + CloudNativeCon Seattle 2018](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)!
+
+
+> Additional KubeCon Resources
+
+> [View All Videos](https://www.youtube.com/playlist?list=PLj6h78yzYM2N8GdbjmhVU65KYm_68qBmo)
+
+> [View Flickr Gallery](https://www.flickr.com/photos/143247548@N03/sets/72157696369579365/with/40046937380/)
+
+
```
Comment (ColinSullivan1) [inline content/blog/kubecon_eu_2018.md]: Can you drop these spaces?

File: content/blog/kubecon_eu_2018.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
++++
+categories = ["Community"]
+date = "2018-05-11"
+tags = ["events", "kubernetes", "kubecon"]
```
Comment (ColinSullivan1) [inline content/blog/kubecon_eu_2018.md]: You may want to add some more like `"CNCF", "cloud", "community-update" , "events", "presentation"`

Comment (gcolliso) [reply content/blog/kubecon_eu_2018.md]: added additional tags

File: content/blog/kubecon_eu_2018.md (hunk 1 above)
Comment (gcolliso) [reply content/blog/kubecon_eu_2018.md]: which empty lines? at the bottom of the doc?

Comment (ColinSullivan1) [reply content/blog/kubecon_eu_2018.md]: Yep.

**Pull request merged**